src/pages/ResetPassword/2-CodeRequest/type.ts

```diff
@@ -14,3 +14,7 @@ export interface ButtonsInfo {
   state: StatesType;
   onPress: () => void;
 }
+
+export interface CodeValidationMessageProps {
+  type: "successful" | "unsuccessful" | string;
```

Comment:
Essa interface tem como melhorar também, esse `| string` no final basicamente anula o "sucessful | unsucessful", já que essa interface não está restringindo ao "sucessful" e "unsucessful", ao invés disso ela ta aceitando qualquer valor que seja uma string.

Isso é algo que tem bastante no nosso código inclusive, typagens que não tem nexo (que eu fiz muito também, to tentando me corrigir). Se você não conseguir arrumar agora, a gente deixa isso pro final da sprint.